src/turbopuffer.ts

```diff
@@ -435,8 +438,8 @@ function fromColumnar(cv: ColumnarVectors): Vector[] {
       vector: cv.vectors[i],
       attributes: cv.attributes
         ? Object.fromEntries(
-            attributeEntries.map(([key, values]) => [key, values[i]]),
-          )
```

Comment:
I'm pretty sure we have 2 different formatter versions fighting each other since mine keeps indenting this

Reply:
can you check this

```
$ npm run prettier --version
10.8.3
```

Reply:
.

Reply:
It's `9.6.7`, but that's the `npm` version

I have prettier 3.3.3, and the version pinned here was different, but that doesn't seem to be what's the discrepancy in my setup. I ran lint:fix here with the prettier here

Might be my tscript lsp or something, not sure

Reply:
yeah I use `npm run format:fix`